rules/Configs/Rector/Closure/ServiceSettersToSettersAutodiscoveryRector.php

```diff
@@ -105,7 +105,7 @@ public function refactor(Node $node): ?Node
         $classNamesAndFilesPaths = $this->createClassNamesAndFilePaths($bareServicesSetMethodCallExpressions);
 
         $classNames = array_map(
-            fn (ClassNameAndFilePath $classNameAndFilePath): string => $classNameAndFilePath->getClassName(),
+            static fn(ClassNameAndFilePath $classNameAndFilePath): string => $classNameAndFilePath->getClassName(),
```

Comment:
Oh, I've missed this in Rector pre-set. I want to have it removed as well, same for closures. It makes code much less readable for me. Thanks :+1:

Reply:
I personally prefer use static, as static closure/arrowfunction is faster, but sure, here it is PR to remove that:

- https://github.com/rectorphp/rector-src/pull/5927